katas/007_Interleave.ipynb

```diff
@@ -0,0 +1,2287 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "13aa221d-f43d-4654-8ec3-117dfdf70ab4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import random\n",
+    "import json\n",
+    "from opensearchpy import OpenSearch\n",
```

Comment:
`opensearchpy` not in `requirements.txt`

Reply:
re-opening: there are two `requirements.txt` files.

One on the top-level of the repo, one in the `katas` folder.

The one in the `katas` folder does not have `opensearch-py` and the installation guide points towards that one, so we should add it there. Adding a comment to the `requirements.txt` file as well.